packages/nouns-contracts/contracts/governance/NounsDAOV3Admin.sol

```diff
@@ -209,29 +206,29 @@ library NounsDAOV3Admin {
      * @notice Admin function for setting the objection period duration
      * @param newObjectionPeriodDurationInBlocks new objection period duration, in blocks
      */
-    function _setObjectionPeriodDurationInBlocks(
-        NounsDAOStorageV3.StorageV3 storage ds,
-        uint32 newObjectionPeriodDurationInBlocks
-    ) external onlyAdmin(ds) {
+    function _setObjectionPeriodDurationInBlocks(uint32 newObjectionPeriodDurationInBlocks) external onlyAdmin {
         if (newObjectionPeriodDurationInBlocks > MAX_OBJECTION_PERIOD_BLOCKS)
             revert InvalidObjectionPeriodDurationInBlocks();
 
-        uint32 oldObjectionPeriodDurationInBlocks = ds.objectionPeriodDurationInBlocks;
-        ds.objectionPeriodDurationInBlocks = newObjectionPeriodDurationInBlocks;
+        uint32 oldObjectionPeriodDurationInBlocks = ds().objectionPeriodDurationInBlocks;
+        ds().objectionPeriodDurationInBlocks = newObjectionPeriodDurationInBlocks;
 
         emit ObjectionPeriodDurationSet(oldObjectionPeriodDurationInBlocks, newObjectionPeriodDurationInBlocks);
     }
 
+    function ds() internal pure returns (NounsDAOStorageV3.StorageV3 storage ds_) {
```

Comment:
prefer to move this function to the bottom of the file

Reply:
sure